mlflow/relations.tf

```diff
@@ -0,0 +1,27 @@
+resource "juju_integration" "mlflow_server_minio_object_storage" {
+  model = juju_model.kubeflow[0].name
+
+  application {
+    name     = module.mlflow_server.app_name
+    endpoint = "object-storage"
+  }
+
+  application {
+    name     = module.minio.app_name
+    endpoint = "object-storage"
+  }
+}
+
+resource "juju_integration" "mlflow_server_mlflow_mysql_client" {
```

Comment:
should we include the endpoint instead of `_client`? I think it's misleading like this, given the previous integration too.